examples/gameroom/daemon/src/rest_api/routes/mod.rs

```diff
@@ -174,44 +174,46 @@ mod tests {
 
     #[test]
     fn test_default_paging_response() {
-        // Create paging response from default limit, default offset, a total of 1000
+        // Create paging response from default limit, default offset, a total of 10000
         let test_paging_response =
-            get_response_paging_info(DEFAULT_LIMIT, DEFAULT_OFFSET, TEST_LINK, 1000);
+            get_response_paging_info(DEFAULT_LIMIT, DEFAULT_OFFSET, TEST_LINK, 10000);
         let generated_paging_response =
-            create_test_paging_response(DEFAULT_LIMIT, DEFAULT_OFFSET, 100, 0, 900);
+            create_test_paging_response(DEFAULT_LIMIT, DEFAULT_OFFSET, 1000, 0, 9000);
         assert_eq!(test_paging_response, generated_paging_response);
     }
 
     #[test]
     fn test_50offset_paging_response() {
-        // Create paging response from default limit, offset of 50, and a total of 1000
-        let test_paging_response = get_response_paging_info(DEFAULT_LIMIT, 50, TEST_LINK, 1000);
-        let generated_paging_response = create_test_paging_response(DEFAULT_LIMIT, 50, 150, 0, 900);
+        // Create paging response from default limit, offset of 500, and a total of 10000
```

Comment:
Is it necessary to update the tests besides `test_default_paging_response()`? If so, the names of the tests should be updated as well